Artifacts/windows-firewall/artifact-funcs.ps1

```diff
@@ -0,0 +1,18 @@
+﻿function Handle-LastError
```

Comment:
This function could be moved into artifact-main.ps1 instead of having it in a separate file. That would match the approach used in https://github.com/Azure/azure-devtestlab/tree/master/Artifacts/windows-iis